muxpi/nuc/ethup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Please add the header in this file as well (the copyright and Description and Usage if applicable).

Reply:
Added